compiler/src/iree/compiler/Dialect/Flow/Transforms/SetEncoding.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The IREE Authors
+// Copyright 2024 The IREE Authors
```

Comment:
I think we want to keep it `2022` because the pass was authored in 2022. We use current year for new files.